### PR TITLE
Complete half baked deliverable

### PR DESCRIPTION
**Pull request by ryanjeffrey**

### CODE REVIEW

File: src/components/Editor/Editor.js [hunk 1]
```diff
@@ -28,7 +33,7 @@ export default function Editor() {
       </div>
       <div className="form-control">
         <label>Alignment</label>
-        <div className="radio-group">
+        <div className="radio-group" onChange={(e) => setAlign(e.target.value)}>
```
Comment (viccaruso) [inline src/components/Editor/Editor.js:36]: Nice use of [event bubbling](https://javascript.info/bubbling-and-capturing#bubbling)!

File: src/components/Home/Home.js [hunk 2]
```diff
@@ -7,12 +7,28 @@ import './Home.css';
...
+        setTitle={setTitle}
+        subtitle={subtitle}
+        setSubtitle={setSubtitle}
+        font={font}
+        setFont={setFont}
+        align={align}
+        setAlign={setAlign}
+        text={text}
+        setText={setText}
+      />
```
Comment (viccaruso) [inline src/components/Home/Home.js]: It doesn't look like you use `align` or `font` in the definition of your `<Editor>` component, so you don't need to pass them here.

**Review by viccaruso — APPROVED** [review]
Looks good to me!

File: src/components/Home/Home.js (hunk 2 above)
Comment (ryanjeffrey) [reply src/components/Home/Home.js]: Awesome, thanks Vic!

**Pull request merged**